templates/registration/edit_profile.html

```diff
@@ -0,0 +1,118 @@
+{% extends 'base.html' %}
```

Comment:
I'd put this file in `templates/user/` to other profile related templates

Reply:
I'd like to suggest also, we change how this form looks like down the line to match the look of profile view.

But right now we should get this form to just work logic-wise

Reply:
Moved to templates/user/

Edited the form to look like Profile form.